packages/ubuntu_wsl_setup/lib/pages/applying_changes/applying_changes_model.dart

```diff
@@ -23,6 +23,11 @@ class ApplyingChangesModel extends SafeChangeNotifier with SystemShutdown {
   bool _previousState = true;
 
   void init({required VoidCallback onDoneTransition}) {
+    if (null != _monitor.status && !_monitor.status!.state.isInstalling) {
+      _previousState = false;
+      onDoneTransition();
```

Comment:
perhaps it was a mistake to use a callback for this... it could be cleaner to return a Future and simply act upon when the call returns :)

Reply:
Indeed. I'll put this into my refactoring wish list.